cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/tools-control.tsx

```diff
@@ -1239,6 +1239,7 @@ export class ToolsControlComponent extends React.PureComponent<Props, State> {
                                     objectType: ObjectType.SHAPE,
                                     frame,
                                     occluded: false,
+                                    rotation: data.rotation == null ? 0 : parseFloat(data.rotation),
```

Comment:
The same comment as for the server, plus why do you use parseFloat here? Does server return string? 
Also ``parseFloat`` for ``undefined`` returns NaN, it will cause issues with models that does not have ``rotation`` in output.  

Here I would suggest: ```rotation: data.rotation || 0``` in case of corresponding shapes.